packages/space-opera/editor/index.html

```diff
@@ -46,6 +46,23 @@
   <script async src='https://www.google-analytics.com/analytics.js'></script>
 </head>
 <style>
+  :root {
```

Comment:
using global colors now, to adjust light/dark theme

Reply:
Global colors is great, but I may have to veto the light theme. First, the two themes are not consistent: lights uses lines for borders while dark uses contrasting backgrounds. Second, having two themes doubles the maintenance for adding features (or if we get a11y feedback), even long after you've moved on from the project.

Reply:
Here are my thoughts on that. I actually think having two themes would increase the long-term productivity of anyone working on this editor moving forward. When introducing a feature, it is extremely easy to get into the habit of copy, pasting a random hex value throughout the code, instead of using a global color. Having a light theme is a reminder that you must use global colors, which I believe makes it easier to keep a unified color scheme throughout the app. In a sense, it forces uniformity, and eases updates in the future. Yes, just having a global color scheme does this as well. But I believe the Light theme forces the global scheme to be used correctly. 

If you still don't think we should have it, it's easy enough to remove, your call. (And I can change the Light Theme cards to have a background contrast instead of border, easy enough as well.)

Reply:
Yeah, agreed on the global colors, but I'm happy to enforce that through code review. Maintaining the Light Theme is still too much maintenance (and review).